### PR TITLE
Fixed unclarity about free users consuming content

### PR DESCRIPTION
**Pull request by marclelijveld**
The original text left the assumption that people can view content with a free license, if a pro user shared it with them. This is only applicable if the content is backed by a premium capacity. Otherwise the consuming user requires the same license as the content developer.

### CODE REVIEW

**Comment by PRMerger6** [comment]
@marclelijveld : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by KesemSharabi** [comment]
Hi @haydnr,

The link should be:

.../enterprise/service-premium-what-is.md#unlimited-content-sharing

In Docs, it'll render as:

https://docs.microsoft.com/power-bi/enterprise/service-premium-what-is#unlimited-content-sharing

(and the local, for example en-us, will be automatically added by your browser according to your system, settings)

Thanks,

Kesen

File: powerbi-docs/consumer/end-user-features.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ LocalizationGroup: consumers
...
+As a Power BI service customer with a Free license, you explore reports and dashboards in order to make business decisions. Those reports and dashboards are created by *designers* who have Power BI Pro or Premium Per User (PPU) licenses and must be put in a workspace backed by a premium capacity in order to benefit from [unlimited content sharing](https://docs.microsoft.com/en-us/power-bi/enterprise/service-premium-what-is?WT.mc_id=DP-MVP-5003435#unlimited-content-sharing) as part of Premium. In case user with a Pro or PPU license share content in a regular workspace, you need an identical license as the *_designers_* have. Pro and PPU users have the ability to share content with their colleagues and to control what their colleagues can and can't do with that content. Sometimes, designers share content by sending you links, and sometimes the content automatically installs and appears in Power BI under **Apps** or **Shared with me**.
```
Comment (maggiesMSFT) [inline powerbi-docs/consumer/end-user-features.md:19]:
```suggestion
As a Power BI service customer with a Free license, you explore reports and dashboards in order to make business decisions. Those reports and dashboards are created by *designers* who have Power BI Pro or Premium Per User (PPU) licenses. The designers need to share content to a workspace backed by a Premium capacity, to benefit from [unlimited content sharing](../enterprise/service-premium-what-is.md#unlimited-content-sharing) as part of Premium. If a designer with a Pro or PPU license shares content in a regular workspace, you need the same license that the *designers* have. Pro and PPU users have the ability to share content with their colleagues and to control what their colleagues can and can't do with that content. Sometimes, designers share content by sending you links, and sometimes the content automatically installs and appears in Power BI under **Apps** or **Shared with me**.
```

**Comment by PRMerger20** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by maggiesMSFT** [comment]
@marclelijveld -- thanks for clarifying!
#sign-off

**Pull request merged**